micronota/rules/config.yaml

```diff
@@ -0,0 +1,29 @@
+# settings of wrapped tools
+tools:
+    prodigal:
+        # params: '-p meta'
+        params: '-p single'
+        priority: 90
+        threads: 1
+    # cmscan:
+    #     params: ''
+    #     priority: 50
+    #     threads: 1
+    #     db: '/Users/zech/Dropbox/git/micronota/micronota/data/RF00169.cm'
```

Comment:
Ok, here's a good solution.

Can you replace all of these paths with relative paths.  One of the most frustrating things about reviewing this code base is guessing the file formats.  Here, I have no idea what RF00169.cm actually is.  If you could actually include that file, the path to that file (in a testing data directory), it'll make this infinitely easier to understand, and speed up the reviews.